mk/containerized.mk

```diff
@@ -36,9 +36,12 @@ push: ## Push nvmeof and nvmeof-cli containers images to quay.io registries
 				echo "Pushing image $(QUAY)/$${image}:$${version}$(TAG_SUFFIX) ...";  \
 				docker tag $(CONTAINER_REGISTRY)/$${image}:$(VERSION)$(TAG_SUFFIX) $(QUAY)/$${image}:$${version}$(TAG_SUFFIX) && \
 				docker push $(QUAY)/$${image}:$${version}$(TAG_SUFFIX); \
-			fi \
-		done \
-	done
+			fi; \
+		done; \
+	done; \
+	echo "Pushing image \"$(QUAY)/spdk:$(SPDK_VERSION)\" ..."; \
```

Comment:
It is better to add '@' before the echo command to prevent it from being printed to the terminal.